src/System.Security.Cryptography.Xml/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard;
+      uap
```

Comment:
I was getting errors that compile time and runtime assemblies are missing - should I remove UAP from the package?